app/plugin/api.go

```diff
@@ -0,0 +1,34 @@
+package plugin
```

Comment:
There should be a test for this: https://mattermost.atlassian.net/browse/PLT-7264